.github/workflows/manifold.yml

```diff
@@ -21,25 +21,54 @@ jobs:
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
     # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
-    - uses: actions/checkout@v2
-    - uses: jwlawson/actions-setup-cmake@v1.12
-    
     - name: Install dependencies
       run: |
         apt-get -y update
-        DEBIAN_FRONTEND=noninteractive apt install -y libglm-dev libassimp-dev libcgal-dev
-    - name: Build CUDA
+        DEBIAN_FRONTEND=noninteractive apt install -y libglm-dev libassimp-dev libcgal-dev libomp-dev nodejs git python3
```

Comment:
If we're going to submodule these, should we skip installing them?

Reply:
We can do this for glm. For assimp, the build script will only compile the submodule if building with emscripten to minimize build time, but we can change that if you like.